editor/src/components/inspector/flex-section.tsx

```diff
@@ -5,24 +5,56 @@ import { Substores, useEditorState } from '../editor/store/store-hook'
 import { AddRemoveLayouSystemControl } from './add-remove-layout-system-control'
 import { FlexDirectionToggle } from './flex-direction-control'
 import { selectedViewsSelector, metadataSelector } from './inpector-selectors'
-import { detectAreElementsFlexContainers } from './inspector-common'
+import {
+  detectAreElementsFlexContainers,
+  detectPackedSpacedSetting,
+  PackedSpaced,
+} from './inspector-common'
 import { NineBlockControl } from './nine-block-controls'
 import { UIGridRow } from './widgets/ui-grid-row'
 import { PaddingRow } from '../../components/inspector/sections/layout-section/layout-system-subsection/layout-system-controls'
+import { SpacedPackedControl } from './spaced-packed-control'
+import { ThreeBarControl } from './three-bar-control'
+import { assertNever } from '../../core/shared/utils'
 
 const areElementsFlexContainersSelector = createSelector(
   metadataSelector,
   selectedViewsSelector,
   detectAreElementsFlexContainers,
 )
 
+const packedFlexSettingSelector = createSelector(
+  metadataSelector,
+  selectedViewsSelector,
+  detectPackedSpacedSetting,
+)
+
 export const FlexSection = React.memo(() => {
   const allElementsInFlexLayout = useEditorState(
     Substores.metadata,
     areElementsFlexContainersSelector,
     'FlexSection areAllElementsInFlexLayout',
   )
 
+  const packedSpacedSetting =
+    useEditorState(
+      Substores.metadata,
+      packedFlexSettingSelector,
+      'FlexSection packedFlexSetting',
+    ) ?? 'packed'
+
+  // TODO: turn this into a display none inside the controls to avoid re-rendering
```

Comment:
For this PR or future work?

Reply:
for this PR, I missed it when I opened up the PR to reviews